Marlin/src/gcode/calibrate/G28.cpp

```diff
@@ -62,13 +62,7 @@
     current_position.set(0.0, 0.0);
     sync_plan_position();
 
-    const int x_axis_home_dir =
-      #if ENABLED(DUAL_X_CARRIAGE)
-        x_home_dir(active_extruder)
-      #else
-        home_dir(X_AXIS)
-      #endif
-    ;
+    const int x_axis_home_dir = x_home_dir(active_extruder);
```

Comment:
Does this line behave like previous? Can't analyze function contents now

Reply:
Yes. There is now a simple pass-through for non-dual-x.